### PR TITLE
Buildah v1.16.0 Release Announcement

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

RA for the v1.16.0 release.

### CODE REVIEW

File: _posts/2020-09-16-Buildah-version-v1.16.0.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  * The /etc/host and /etc/resolv.conf files are no longer bound if network is not present in the container.
+  * An unnecessary call to the function NewImage() has been removed from the build processing.
+  * When processing multiple archives during a .dockerignore process, the processing would stop at the end of the first archive.  Now all archives are processed as they should be.
+  * Fixed & added notes regarding problematic language in the codebase that were not inclusive.
+  * Added a dependency on github.com/stretchr/testify/require.
+  * The build processing now waits for stages that might not have even started yet instead of trying to continue and then failing.
+  * Mounts under `/sys` were not always accessible to rootless users.  This has been corrected.
+  * Environment variables can now be pre-declared in the containers.conf(5) file.
+  * The right stage's image is now returned as the "final" image.
+  * When build arguments and environment variables have duplicate names, the values are now deduplicated.  Build arguments override Default arguments and Environment variables set during the build process overrides both of those.
```
Comment (TomSweeneyRedHat) [inline _posts/2020-09-16-Buildah-version-v1.16.0.md:55]: @nalind can you triple check that I have this ordering correct here, please?

Comment (nalind) [reply _posts/2020-09-16-Buildah-version-v1.16.0.md:55]: Yes, that's the correct ordering.  That last "overrides" should be "override" (subject-verb agreement).

Comment (TomSweeneyRedHat) [reply _posts/2020-09-16-Buildah-version-v1.16.0.md:55]: TY and tweaked.

**Comment by TomSweeneyRedHat** [comment]
Just repushed with @nalind 's change and also a private comment from @jwhonce

**Pull request merged**